.github/workflows/build-and-test-dotnet.yml

```diff
@@ -115,7 +121,7 @@ on:
       dotnet-consolidation-exclude-version-regex:
         type: string
         required: false
-        default: "-alpha"
+        default: '-alpha'
```

Comment:
Why this change? When using single quotes, the samples in the descriptions (like <<When set to "true">>) should also use single quotes.

Also in the other workflows.

Related to https://github.com/Lombiq/NodeJs-Extensions/issues/11.

Reply:
It was autoformatting (`"yaml.format.singleQuote": true,` setting helps with this for VS Code) to match all the other parameters. Examples values in descriptions are surrounded by doublequotes (at least in the workflows I modified, so no change needed there). There are a few cases of backticks, but that just makes harder to type/read, because it's not rendered.

Reply:
I'm OK with the single quote style (since in `if`s it's required anyway; though changing everything instead of just certain parameters would be better) but if we say that strings should be surrounded with single quotes then examples of strings should use that too.

Reply:
I intended to go through each file and add fix formatting for every file, but that would be a lot of stuff in the diff that is not relevant to this PR, so I kept it within the boundaries of the files I'm working on.

Since we're working on automating YAML linting, we'll have to change a lot of stuff soon anyway.

However, default values (for workflows, jobs, parameters) don't need any surrounding apostrophes or double quotes at all, so we should get rid of those completely (and enforce this).

Text intended for humans only (i.e., not processed/rendered) should use double quotes.

Reply:
Yeah, I wouldn't want all such styling changes in this PR either.